src/DynamoCoreWpf/Properties/AssemblyInfo.cs

```diff
@@ -58,10 +58,13 @@
 [assembly: InternalsVisibleTo("DynamoPlayer")]
 [assembly: InternalsVisibleTo("DynamoConnector")]
 
-
+// Disable PublicAPIAnalyzer errors for this type as they're already added to the public API text file
+#pragma warning disable RS0016 
```

Comment:
so even if these types are added to the unshipped list we still get errors?

Reply:
Yes, I'm not sure why this is happening.